lesson_02_homework.rb

```diff
@@ -16,7 +18,8 @@ def full_name(first_name, last_name, title)
   else
     raise "Oh no, that doesn't look like a name"
   end
-
+  # Above example is called 'Conditional statement' or 'If statement'
+  # and excutes program acccording to conditions of the statement.
   return name
 end
 
```

Comment:
🔨 @Alisher778 the parameter names should be more explicit `f` and `l` do not describe what type of information the variable is expected to hold